test/plugin/scenarios/elasticsearch-7.x-scenario/src/main/java/org/apache/skywalking/apm/testcase/elasticsearch/RestHighLevelClientCase.java

```diff
@@ -97,27 +100,37 @@ public String elasticsearch() throws Exception {
 
             // create
             createIndex(indexName);
+            createIndexAsync(indexName2);
 
             // index
             index(indexName);
+            indexAsync(indexName2);
 
             // refresh
             client.indices().refresh(new RefreshRequest(indexName), RequestOptions.DEFAULT);
+            Thread.sleep(1000);
```

Comment:
Do we have another way to verify the request has been verified? 
Such as, do it through query or callback?

Reply:
`Sleep` usually is not stable when doing this, when resource is limited in GHA scenario.

Reply:
OK, I will improve that.